tests/functional/utilities/reportHandler.js

```diff
@@ -136,7 +139,32 @@ describe('reportHandler::getCRRStats', function testSuite() {
     this.timeout(20000);
     let redisClient;
 
-    before(done => {
+    function assertResults(res) {
```

Comment:
minor nitpick: imo helper functions should be at the beginning of file/outside of the test functions themselves